$OpenBSD: patch-bindings_python_build_py,v 1.1 2012/05/03 08:48:26 landry Exp $
paths.py ends up with an insane ../../../ garbage pointing to /local otherwise
--- bindings/python/build.py.orig	Thu May  3 00:00:31 2012
+++ bindings/python/build.py	Thu May  3 00:03:21 2012
@@ -143,10 +143,7 @@ paths += "__all__ = [mapniklibpath,inputpluginspath,fo
 if not os.path.exists('mapnik'):
     os.mkdir('mapnik')
 
-if hasattr(os.path,'relpath'): # python 2.6 and above
-    file('mapnik/paths.py','w').write(paths % (os.path.relpath(env['MAPNIK_LIB_DIR'],target_path)))
-else:
-    file('mapnik/paths.py','w').write(paths % (env['MAPNIK_LIB_DIR']))
+file('mapnik/paths.py','w').write(paths % (env['MAPNIK_LIB_DIR']))
 
 # force open perms temporarily so that `sudo scons install`
 # does not later break simple non-install non-sudo rebuild
